src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+
+Too many files, especially large media and large complex scripts must get retrieved, read, processed, and rendered by the browser, increase the chance that pages will take so long to load that users will abandon them. Large payloads can also overwhelm the amount of client side resources available on the users smartphone or computer causing it to stall and even crash the device. Users who have the good fortune to subscribe to high speed cable internet services, or 5G data plans will seldom experience these problems. But again, a large percentage of internet users don't have access to those levels of internet services and devices.
+
+### Transmission
+Transmission is basically the process of transferring a file from one computer to another through one of several communication methods or channels. When a user requests a web page via HTTP(S), all files needed to render the page are requested. Files are located and sent back to the requesting device using a transport protocol and, if all goes well, the requester's browser will take the payload, and process and render it as part of the larger web page on the requesting user's screen. Page weight becomes important during the transmission process because the size of the file determines how long it will take to complete the transfer, and how long and how many resources will be needed to render the results.
+
+Another negative side effect of large page weight is increased [latency](https://developer.mozilla.org/en-US/docs/Web/Performance/Understanding_latency) (the time it takes for the request to connect to the server storing the files and begin the process of transporting those files). If bunches of files are requested, no matter the technology, there is a limit on how much can be processed and transferred in any given period. I've audited WordPress sites that request as many as 170 files or more, which ensures terrible page loading performance starting with high latency periods.
+
+There are certainly optimizations that can be done, such as compressing and combining certain file requests, using HTTP/2 or the newer HTTP/3 protocols, and using a modern browser's ability to preconnect to and preload certain files to speed the the whole process process up. The [Performance chapter](https://almanac.httparchive.org/en/2021/performance) covers a wide range of factors that effect page loading performance.
+
+## Assets
+As explained in last years chapter, there's been little change in most assets in terms of technology, but there are some notable exceptions.
+
+### Static
+Static files reside by themselves and are used as resources to help build out and render web pages like video and audio files, and font files are all examples of static assets. [Images](https://almanac.httparchive.org/en/2021/media), which make a large percentage of the average web page's weight, are also a type of static asset file. So, let's use images for our example.
```

Comment:
```suggestion
Static files reside by themselves and are used as resources to help build out and render web pages. Image, video, audio, and font files are all examples of static assets. [Images](https://almanac.httparchive.org/en/2021/media) make a large percentage of the average web page's weight, so let's use them for our example.
```

I'm confused what the example is for. Or maybe this section? Are you just describing what each asset type is, or were you going to talk about changes from last year to this year? Or both? It's up to you, I'm just saying it might be a little unclear to the reader what the purpose is here.

Consider putting the actual % number for images (as a total of page weight) here as a teaser.